sherpa/astro/ui/utils.py

```diff
@@ -12401,13 +12409,19 @@ def save_all(self, outfile=None, clobber=False):
         Notes
         -----
 
-        Items which are not saved include:
+        This command will create a series of commands that restores
+        the current Sherpa set up. It does not save the set of commands
+        used. Not all Sherpa settings are saved. Items not fully restored
+        include:
 
-        - user models
+        - data created by calls to `load_arrays`, or changed from the
```

Comment:
the docstring to save_all says that data created with calls to load_arrays is not saved, but that seems to be actually be the case, both by reading the code and by performing a simple test. Am I missing something?